tests/startup/test_cli.py

```diff
@@ -151,3 +153,60 @@ def test_enterprise_disabled(self):
                 ''')
             # MQTT
             self.assert_mqtt_port(lambda x: x > 0)
+
+    def test_startup_logs(self):
+
+        # Create CRATE_HOME directory
+        tmp_home = self.mkdtemp()
+        Path(tmp_home, 'config').mkdir()
+
+        nodeName = self.fake.last_name()
+        settings = dict({
+            'node.name': nodeName,
+            'path.home': tmp_home,
+        })
+
+        logFilePath = Path(tmp_home, 'crate.log')
+        # Copy log4j2.properties and replace path to crate.log output file
+        with open(Path(Path(__file__).parent, 'log4j2_appendtofile.properties'), 'r') as f:
+            newlines = []
+            for line in f.readlines():
```

Comment:
It's possible to iterate directly on the file object to avoid the list creation done by calling `readlines`. `for line in f:`